samples/java-spring-shopping-cart-quickstart/.bundleignore

```diff
@@ -0,0 +1,3 @@
+target
```

Comment:
what is .bundleignore?

Reply:
This is needed to exclude some files when creating the zip, like in https://github.com/lightbend/kalix-jvm-sdk/pull/1537/files#diff-cdec797dddf8fc387304501d6bd4f318e1e9067c6fae012dfa8b69fd85fd3248R93